elements/xml_form_elements.module

```diff
@@ -300,6 +300,17 @@ function xml_form_elements_form_element_tabs_ajax_add(FormElement $element, arra
   $new_tab = clone $tab;
   $new_tab->eachDecendant(function($element) {
     $element->default_value = $element->getOriginalDefaultValue();
+                            if ($element->controls['#type'] == 'tags') {
```

Comment:
@knjackson30974

Did you try to move (L302)[https://github.com/Islandora/islandora_xml_forms/pull/212#discussion-diff-52355868R302] to match the indentation of the rest of the block?

If it fails to take that then I advise a comment in the code explaining the coder hack.

Thanks!

----
William Panting
_Developer_
*[discoverygarden inc.](http://www.discoverygarden.ca) | Managing Digital Content*